recipes/emsdk/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.18.2)
```

Comment:
In ConanCenter we will converge to the minimum requirements for Conan v2, to maximize compatibility of recipes with the next Conan versions. Requirement for Conan v2 is CMake 3.15 (https://github.com/conan-io/tribe/blob/main/design/004-tools-cmake.md), so we try to ensure our recipes works with it.

Here I don't see anything requiring that CMake version, so it is probably safe to use something lower